app/GroupMember.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class GroupMember extends Model
+{
+    public  $timestamps = false;
+
+    /**
+     * The table associated with the model.
+     *
+     * @var string
+     */
+    protected $table = 'groupmembers';
+
+    protected $fillable = [
+        'group_id'
+        ,'username'
+        ,'status_code'
+        ,'action_group_id'
```

Comment:
In our current database, the action_group_id column is included. But i agree with Matt, this is not needed based on our current flow. Unless @vincent-h-lee has something in mind for this one.